ibis/backends/dask/tests/execution/test_operations.py

```diff
@@ -84,13 +84,13 @@ def test_project_scope_does_not_override(t, df):
     tm.assert_frame_equal(result.compute(), expected.compute())
 
 
-@pytest.mark.xfail(reason="TODO - aggregations - #2553")
 @pytest.mark.parametrize(
     'where',
     [
         lambda t: None,
-        lambda t: t.dup_strings == 'd',
-        lambda t: (t.dup_strings == 'd') | (t.plain_int64 < 100),
+        # TODO - aggregations - #2553
+        # lambda t: t.dup_strings == 'd',
```

Comment:
you can mark these xfail with `pytest.param([0, 1, 2, pd.NA], True, marks=pytest.mark.xfail),` for example (obvious put the params in)

Reply:
we can also enable (or maybe already?) strict=True, so when these are fixed the xpasses will fail the build